### PR TITLE
[13.0][ADD] uom module

### PR DESCRIPTION
**Pull request by kos94ok-3D**
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]
Make sense. Thanks !

File: addons/uom/migrations/13.0.1.0/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---Models in module 'uom'---
+---Fields in module 'uom'---
+uom          / uom.category             / measure_type (selection)      : selection_keys is now '['length', 'unit', 'volume', 'weight', 'working_time']' ('['length', 'time', 'unit', 'volume', 'weight']')
+# Done in post-migration script
```
Comment (pedrobaeza) [inline addons/uom/migrations/13.0.1.0/openupgrade_analysis_work.txt]: You have to rename the variable to `None` in pre and then map on post. And remember the format for these annotations I talked about in the other issue.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: addons/uom/migrations/13.0.1.0/openupgrade_analysis_work.txt (hunk 1 above)
Comment (kos94ok-3D) [reply addons/uom/migrations/13.0.1.0/openupgrade_analysis_work.txt]: Is it ok?

**Comment by pedrobaeza** [comment]
Sorry, I told you wrong... We need to copy column, not to rename it.

**Comment by pedrobaeza** [comment]
Let me explain the why of this for not getting the impression of picky choice: if you let the incorrect value, any ORM operation writing that records can break the update process due to the incorrect selection value. About the copying process instead of direct mapping, it's for not losing the original value. In this case, being a direct renaming, it's not very critical, but in other cases a value is mapped to other existing value, and there's no way of differentiating between the original ones and the recently mapped ones.

I have also told you bad that the mapping is on post. It must be on pre. Sorry for not being very focused these days.

**Comment by kos94ok-3D** [comment]
@pedrobaeza 
What about it?
изображение

**Comment by pedrobaeza** [comment]
Not in this case as explained above about the risk, and there's nothing counter-productive in doing it in pre.

**Comment by kos94ok-3D** [comment]
@pedrobaeza Done

File: addons/uom/migrations/13.0.1.0/openupgrade_analysis_work.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---Models in module 'uom'---
+---Fields in module 'uom'---
+uom          / uom.category             / measure_type (selection)      : selection_keys is now '['length', 'unit', 'volume', 'weight', 'working_time']' ('['length', 'time', 'unit', 'volume', 'weight']')
+# DONE: pre-migration: renamed `measure_type` to None then mapped 'time' value to 'working_time'
```
Comment (pedrobaeza) [inline addons/uom/migrations/13.0.1.0/openupgrade_analysis_work.txt]:
```suggestion
# DONE: pre-migration: copy `measure_type` and then mapped 'time' value to 'working_time'
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by StefanRijnhart** [comment]
Not sure why the Travis job is not marked as success for the gitbot PRs

/ocabot merge

**Pull request merged**